File: src/main/java/no/nav/service/pensjon/regler/to/BeregnAfpPrivatRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.ArrayList;
import java.util.Date;

import no.nav.domain.pensjon.regler.beregning2011.BeregningsResultatAfpPrivat;
import no.nav.domain.pensjon.regler.krav.Kravhode;
import no.nav.domain.pensjon.regler.vedtak.VilkarsVedtak;

public class BeregnAfpPrivatRequest extends ServiceRequest  { 

	private Kravhode kravhode;
	private ArrayList<VilkarsVedtak> vilkarsvedtakListe;
	private Date virkFom;
	private double ft;
	private int justeringsbelop;
	private int referansebelop;
	private double ftKompensasjonstillegg;
	private BeregningsResultatAfpPrivat sisteAfpPrivatBeregning;
	private Date virkFomAfpPrivatUttak;

}

File: src/main/kotlin/no/nav/pensjon/regler/to/BeregnAfpPrivatRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.beregning2011.BeregningsResultatAfpPrivat
import no.nav.pensjon.regler.domain.krav.Kravhode
import no.nav.pensjon.regler.domain.vedtak.VilkarsVedtak
import java.util.*
import java.time.LocalDate

class BeregnAfpPrivatRequest : ServiceRequest() {
    var kravhode: Kravhode? = null
    var vilkarsvedtakListe: ArrayList<VilkarsVedtak> = ArrayList()
    @Deprecated("Use virkFomLd instead")
    var virkFom: Date? = null
    var virkFomLd: LocalDate? = null
    var justeringsbelop = 0
    var referansebelop = 0
    var ftKompensasjonstillegg = 0.0
    var sisteAfpPrivatBeregning: BeregningsResultatAfpPrivat? = null
    @Deprecated("Use virkFomAfpPrivatUttakLd instead")
    var virkFomAfpPrivatUttak: Date? = null
    var virkFomAfpPrivatUttakLd: LocalDate? = null
}